android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/LoginScreen.kt

```diff
@@ -390,6 +391,7 @@ fun LoginPage(
 
 @Composable
 fun ForgotPasswordDialog(
+  applicationConfiguration: ApplicationConfiguration,
```

Comment:
Pass the contact number instead of providing the `ApplicationConfiguration` entirely. That way the composable function will only need to watch for changes in the contact number but not the entire config data class.